webapp/channels/src/components/interactive_dialog/interactive_dialog.tsx

```diff
@@ -87,7 +100,7 @@ export default class InteractiveDialog extends React.PureComponent {
 
         this.setState({errors});
 
-        if (Object.keys(errors).length !== 0) {
+        if (Object.keys(errors!).length !== 0) {
```

Comment:
Why you need here the `!`, looks like is not needed, errors shouldn't be null in any case here.

Reply:
Yess. ! But ts was not detecting it and it was giving error : (